code/datums/supplypacks.dm

```diff
@@ -2888,3 +2888,12 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
 	containertype = /obj/structure/stackopacks
 	containername = "\improper Meat Fridge stack of packs"
 	group = "Vending Machine packs"
+
+/datum/supply_packs/art
+	name = "Art Supply stack of packs"
+	contains = list(/obj/machinery/vending/art,
```

Comment:
I really should have caught this before but someone reported it to me just now. These are supposed to contain resupply packs, not actual vendors.

Reply:
oh mein gott